site/en/blog/media-updates-in-chrome-69/index.md

```diff
@@ -0,0 +1,289 @@
+won't have any applications to handle that intent.
+
+To fix this, Chrome 69 on Android Go now listens for media-viewing intents so
+users can view downloaded audio, videos, and images. In other words, it takes
+the place of the missing viewing applications.
+
+<figure>
+  {% Img src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/qGcEJcsQz6oRpoqE8mCE.png", alt="ALT_TEXT_HERE", width="800", height="369" %}
+  <figcaption>
+    Media intent handler
+  </figcaption>
+</figure>
+
+Note that this Chrome feature is enabled on all Android devices running Android
+O and onwards with 1 GB of RAM or less.
+
+[Chromium Bug](https://crbug.com/718641)
+
+## Removal of "stalled" events for media elements using MSE {: #stalled }
+
+A "stalled" event is raised on a media element if downloading media data has
+failed to progress for about 3 seconds. When using [Media Source Extensions
+(MSE)], the web app manages the download and the media element is not aware of
+its progress. This caused Chrome to raise "stalled" events at inappropriate
+times whenever the website has not appended new media data chunks with
+`SourceBuffer.appendBuffer()` in the last 3 seconds.
+
+As websites may decide to append large chunks of data at a low frequency, this
+is not a useful signal about buffering health. Removing "stalled" events for
+media elements using MSE clears up confusion and brings Chrome more in line
+with the MSE specification. Note that media elements that don't use MSE will
+continue to raise "stalled" events as they do today.
+
+[Intent to Deprecate and Remove](https://groups.google.com/a/chromium.org/d/msg/blink-dev/x54XtrTyOP8/4-5QZlZzDAAJ) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/6338037575319552) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=836951)
+
+<!-- lint disable definition-case -->
+
+[Chrome 70]: /blog/media-updates-in-chrome-70#av1-decoder
+[CENC mode]: https://www.iso.org/obp/ui/#iso:std:iso-iec:23001:-7:ed-2:v1:en
+[CBCS mode]: https://www.iso.org/obp/ui/#iso:std:iso-iec:23001:-7:ed-3:v1:en
+[added]: https://github.com/WICG/encrypted-media-encryption-scheme/blob/main/explainer.md
+[dictionary]: https://w3c.github.io/encrypted-media/#idl-def-mediakeysystemmediacapability
+[Encrypted Media Extensions (EME)]: https://w3c.github.io/encrypted-media/
+[Clear Key]: https://www.w3.org/TR/encrypted-media/#clear-key
+[HDCP]: https://en.wikipedia.org/wiki/High-bandwidth_Digital_Content_Protection
+[protected content]: https://developers.google.com/web/fundamentals/media/eme
+[HDCP Policy Check API]: https://github.com/WICG/hdcp-detection/blob/main/explainer.md
+[other error values]: https://w3c.github.io/encrypted-media/#dom-mediakeystatus
+[official sample]: https://googlechrome.github.io/samples/hdcp-detection/
+[Origin Trial]: https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md
+[request a token]: http://bit.ly/HdcpPolicyCheckOriginTrials
```

Comment:
remove http:// and replace with https://

Reply:
Done